sdk/storage/storage-blob-changefeed/src/utils/utils.common.ts

```diff
@@ -193,7 +193,7 @@ export function rawEventToBlobChangeFeedEvent(rawEvent: any): BlobChangeFeedEven
         delete previousInfo.SoftDeleteSnapshot;
       }
       if (previousInfo.WasBlobSoftDeleted) {
-        previousInfo.isBlobSoftDeleted = previousInfo.WasBlobSoftDeleted;
+        previousInfo.isBlobSoftDeleted = previousInfo.WasBlobSoftDeleted === "true";
```

Comment:
Do you need to care about case of string "true"?

Reply:
It has been transferred to a boolean, so I just need to check whether value for `previousInfo.isBlobSoftDeleted` is correct, which won't need to change current case.